CC/Sounder/receiver.cc

```diff
@@ -435,6 +524,7 @@ void Receiver::clientSyncTxRx(int tid)
                 rx_offset = 0;
                 if (resync) {
                     sync_index = CommsLib::find_beacon_avx(buff0, config_->gold_cf32);
+                    std::cout << "frame_cnt: " << frame_cnt << ", sync_index: " << sync_index << std::endl;
```

Comment:
This line causes a lot of prints at the end. Can we remove it?

Reply:
or make it a debug print?

Reply:
Done, remove!